File: ArrayManipulation/array.java
import java.util.Arrays;
import java.util.Collections;
import java.util.List;
import java.util.ArrayList;

public class ArrayManipulation {
    public static void main(String[] args) {
        List<Integer> numbers = new ArrayList<>(Arrays.asList(10, 25, 8, 42, 15, 33, 7));
        
        // Calculate sum
        int sum = numbers.stream().mapToInt(Integer::intValue).sum();
        
        // Calculate average
        double average = numbers.stream().mapToInt(Integer::intValue).average().orElse(0.0);
        
        // Find maximum
        int max = Collections.max(numbers);
        
        System.out.println("Array: " + numbers);
        System.out.println("Sum: " + sum);
        System.out.println("Average: " + average);
        System.out.println("Maximum: " + max);
    }
}

File: ArrayManipulation/array.c
#include <stdio.h>

int main() {
    int numbers[] = {10, 25, 8, 42, 15, 33, 7};
    int size = sizeof(numbers) / sizeof(numbers[0]);
    
    // Calculate sum
    int sum = 0;
    for (int i = 0; i < size; i++) {
        sum += numbers[i];
    }
    
    // Calculate average
    double average = (double)sum / size;
    
    // Find maximum
    int max = numbers[0];
    for (int i = 1; i < size; i++) {
        if (numbers[i] > max) {
            max = numbers[i];
        }
    }
    
    printf("Array: ");
    for (int i = 0; i < size; i++) {
        printf("%d ", numbers[i]);
    }
    printf("\n");
    printf("Sum: %d\n", sum);
    printf("Average: %.2f\n", average);
    printf("Maximum: %d\n", max);
    
    return 0;
}